### ISSUE TITLE
Sticky "invalid admin password" notification is not removed after a successful login

### ISSUE THREAD

**Issue by dev7355608** [foundryvtt/foundryvtt#11439]
Originally reported by manaflower: https://discord.com/channels/170995199584108546/1184176415139184731/1260302775909351495

**ALL MODULES DISABLED?** true
**OS, Hosting, Browser (if applicable)**: any
**Short Description of bug**: Failing admin login leaves a sticky error notification about it.
**Simple steps to reproduce the bug**:
1) Purposefully mistype your admin password.
2) Get it right the second time.
3) Observe how you still have the red warning about wrong password after logging in (assuming you didn't manually dismiss it)
**Screenshots and/or console errors**:
---
This bug existed in v11 too.

**Comment by stwlam** [comment]
~~Nonrepro, and I don't see how this is even possible since there's a browser navigation from `/join` to `/game`. Perhaps a relic of V12?~~ ope

**Comment by alexdiste** [comment]
it happened to me yesteday in v14, I put a wrong administration password and press enter, of course fail autentication then insert the right one and I received the error telling password invalid. Maybe is a race condition where notification are sliglihy delayed and so showed after a sucessful login.

**Comment by dev7355608** [comment]
@stwlam I think the report about is about the brief moment both notifications are displayed until the redirect to `/game` happens. 

<img width="1621" height="520" alt="Image" src="https://github.com/user-attachments/assets/7cf67738-b194-4aef-b865-537a02415500" />

**Comment by roth-michael** [comment]
I read this as an issue not when joining a game, but when entering the admin password for the server instance itself

<img width="1792" height="365" alt="Image" src="https://github.com/user-attachments/assets/663d7bbb-c80e-4739-be48-88265c783498" />

**Comment by mkahvi** [comment]
> I think the report about is about the brief moment both notifications are displayed until the redirect to /game happens.

No, it was specifically about admin pass logging into the setup screen. That's why admin password was spoken of. So I hope this got fixed in the right place. The report is fairly old by now so the behaviour might've changed between the report and now.
